src/utils/__tests__/makeReactNativeConfig.test.js

```diff
@@ -13,6 +13,9 @@ import {
 } from '../makeReactNativeConfig';
 
 test('creates config from defaults', () => {
+  // We need to go one level higher because of read polyfills with fs.readFileSync
+  process.chdir(`${__dirname}/..`);
```

Comment:
Undo it after the test finishes

Reply:
Also, `path.join(__dirname, '..');` to be fair on Windows

Reply:
Yes i forgot about support Win 😞